file-formats/apic/type/zif_aff_apic_v1.intf.abap

```diff
@@ -0,0 +1,17 @@
+INTERFACE zif_aff_apic_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">API Catalog Properties</p>
+    "! API Catalog properties
```

Comment:
I think since `API catalog` is no product name we can use just sentence style here, don't we?
```suggestion
    "! API catalog properties
```

Reply:
I have just double-checked it with one of our user assistance developers. My suggestion above is valid.